examples/sudo.schema.ldif

```diff
@@ -0,0 +1,72 @@
+dn: cn=sudo,cn=schema,cn=config
```

Comment:
Absolutely, this is just example material meant to bring up a sandbox environment and should be considered / presented as such.

I can remove it but I think it can be helpful for somebody who wants to try out monban without too much commitment (eg. using the instructions in the README). 

Also maybe this will be useful in an end-to-end test scenario should such tests get implemented sometime.